materialize-bigquery/bigquery.go

```diff
@@ -391,13 +388,5 @@ func (c client) InstallFence(ctx context.Context, _ sql.Table, fence sql.Fence)
 	fence.Fence = bqFence.Fence
 	fence.Checkpoint = checkpoint
 
-	log.WithFields(log.Fields{
-		"fence":            fence.Fence,
-		"keyBegin":         fence.KeyBegin,
-		"keyEnd":           fence.KeyEnd,
-		"materialization":  fence.Materialization.String(),
-		"checkpointsTable": fence.TablePath,
-	}).Info("fence installed successfully")
-
```

Comment:
I find these kinds of logs useful, perhaps keep them around as `Debug` logs?

Reply:
I'll keep these ones around installing & reading the fence around since otherwise it is kind of weird when the last thing logged is "installing fence", which can sometimes need to wait for a long time if a previous commit had a lock on the checkpoints table.